### PR TITLE
add package: perl-rename

### PR DESCRIPTION
**Pull request by ELWAER-M**

### CODE REVIEW

File: packages/perl-rename/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+TERMUX_PKG_SHA256=2f03f33d3a17d3a3599c83f514f0694fd833b606920132925e3fdbd8a1a3e44b
+TERMUX_PKG_DEPENDS="perl"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_configure() {
+    perl Makefile.PL PREFIX=$TERMUX_PREFIX
+}
+
+termux_step_post_massage() {
+    find $TERMUX_PKG_MASSAGEDIR -type f -name rename* -execdir sh -c 'mv {} $(echo {} | sed "s|rename|perl-rename|")' \;
```
[the remark this answers is not in the document]
Comment (ELWAER-M) [reply packages/perl-rename/build.sh]: done

**Comment by ELWAER-M** [comment]
ok i changed them

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Package contains an essentially empty folder `/data/data/com.termux/files/usr/lib/x86_64-linux-gnu/`, it should be removed

File: packages/perl-rename/build.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+TERMUX_PKG_SHA256=2f03f33d3a17d3a3599c83f514f0694fd833b606920132925e3fdbd8a1a3e44b
+TERMUX_PKG_DEPENDS="perl"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_configure() {
+	perl Makefile.PL PREFIX=$TERMUX_PREFIX
+}
+
+termux_step_post_massage() {
+	find $TERMUX_PKG_MASSAGEDIR -type f -name "rename*" -execdir sh -c 'mv {} $(echo {} | sed "s|rename|perl-rename|")' \;
```
[the remark this answers is not in the document]
Comment (ELWAER-M) [reply packages/perl-rename/build.sh:16]: its not for the binary only its also for the manual pages too, i think its better to do that in just one command

[the remark this answers is not in the document]
Comment (Grimler91) [reply packages/perl-rename/build.sh:16]: Good point, thanks for pointing that out

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**